examples/keymanager/keymanager_container/base-key.tf

```diff
@@ -0,0 +1 @@
+../keymanager_secret/main.tf
```

Comment:
do we really need keymanager part in the lowest folder name?

Reply:
no we do not, fixed